c7n/resources/ecr.py

```diff
@@ -69,3 +80,76 @@ def _augment(r):
             resources = list(filter(None, w.map(_augment, resources)))
 
         return super(ECRCrossAccountAccessFilter, self).process(resources, event)
+
+
+@actions.register('remove-statements')
+class RemovePolicyStatement(BaseAction):
+    """Action to remove policy statements from ECR
```

Comment:
as currently implemented this fails for named statements (since it never fetches the policy) and assumes use of cross-accounts filters as a pre-requisite.